calm/draft/2024-10/meta/flow-requirement.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/draft/2024-10/meta/flow-requirement.json",
+  "title": "Generic Flow Specification",
+  "description": "Defines the requirements for a flow",
+  "type": "object",
+  "properties": {
+    "requirementId": {
+      "type": "string",
+      "description": "Unique identifier for the requirement"
+    },
+    "requirementDescription": {
+      "type": "string",
+      "description": "Detailed description of the requirements for the flow"
+    },
+    "fields": {
+      "type": "array",
+      "description": "List of fields required for the flow",
+      "items": {
+        "type": "object",
+        "properties": {
+          "fieldName": {
+            "type": "string",
+            "description": "Name of the field"
+          },
+          "fieldType": {
+            "type": "string",
+            "enum": ["string", "integer", "boolean", "date"],
```

Comment:
Haven't seen this in JSON schema before. I wonder if this would be better modelled by `OneOf`

Reply:
Discussed with @jpgough-ms and confirmed that "type": "string" should not exist at same time as enum